src/pages/index.js

```diff
@@ -42,6 +34,15 @@ const cardFormValidator = new FormValidator(
 );
 cardFormValidator.enableValidation(); // call enable validation
 
+// new avatar form validator instance
+const avatarFormValidator = new FormValidator(
+  settings,
+  document.querySelector("#avatar-form")
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms